string gripper_link
string object_link
int8 command